index.html

```diff
@@ -2,70 +2,59 @@
-
-                <li> <a href="#"> Features </li> </a>
-
-                <li> <a href="#"> Community </li> </a>
-
-                <li> <a href="#"> Blog </li> </a>
-
-                <li> <a href="#"> Pricing </li></a>
-
-
+                <li><a href="#">Home </a></li>
+                <li><a href="#">Features</a></li>
+                <li><a href="#">Community</a></li>
+                <li><a href="#">Blog</a></li>
+                <li><a href="#">Pricing</a></li>
             </ul>
-            <a href="#" class="buttom"> Register Now <img src="./src/right.svg" alt="RightArrow"
-                    style="vertical-align:middle;" width="10" height="12"></a>
-        </nav>
-    </header>
-    <main>
-    <div id="allBlock">
-        <div id="allBlock1">
-            <div id="block1">
-                <h1 class="les">Lessons and insights</h1>
-                <h1 id="from">from 8 years</h1>
-                <p class="greytext">Where to grow your business as a photographer: site or social media?</p>
-                <a href="#" id="reg"> Register</a>
-            </div>
-            <img src="./src/image.png" alt="man" id="man">
+            <a href="#" class="button">Register Now <img src="./src/rightArrow.svg" alt="RightArrow"
+                    class="RightArrow"></a>
+
         </div>
     </div>
-        <div id="dot">
-            <img src="./src/dot1.svg" alt="dot">
-            <img src="./src/dot2.svg" alt="dot">
-            <img src="./src/dot2.svg" alt="dot">
-        </div>
+    <div class="block1">
         <div>
-            <div id="OurClients">
-                <h1 class="les" style="margin-bottom: 0%;"> Our Clients </h1>
-                <p class="greytext" style="margin-top: 0%;"> We have been working with some Fortune 500+ clients </p>
-            </div>
-            <div id="logos">
-                <img src="./src/logos clients/image.png" alt="logos" class="logos">
-                <img src="./src/logos clients/image copy.png" alt="logos" class="logos">
-                <img src="./src/logos clients/image copy 2.png" alt="logos" class="logos">
-                <img src="./src/logos clients/image copy 3.png" alt="logos" class="logos">
-                <img src="./src/logos clients/image copy 4.png" alt="logos" class="logos">
-                <img src="./src/logos clients/image copy 5.png" alt="logos" class="logos">
-                <img src="./src/logos clients/image copy 2.png" alt="logos" class="logos">  
-            </div>
-            <div></div>
+            <h2> Lessons and insights <br> <span> from 8 years </span> </h2>
+            <p class="Subtext">Where to grow your business as a photographer: site or social media?</p>
+            <a href="#" class="button">Register</a>
         </div>
-    </main>
-
-    <footer></footer>
+        <img src="/src/imgBlock1.png" alt="imgBlock1" class="imgBlock1"> 
+    </div>
+    <div class="dots">
+        <img src="./src/dot1.svg" alt="dot1">
```

Comment:
Made it by span or div, not svg

Reply:
![image](https://github.com/user-attachments/assets/3fa81b6f-74d5-4049-8f88-9a0e5d612912)
![image](https://github.com/user-attachments/assets/9e3a266c-8e80-48fe-8e90-8613a630e4ce)